src/main/java/mil/dds/anet/beans/Position.java

```diff
@@ -224,10 +226,30 @@ public Location getLocation() {
         });
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> loadAllPeoplePositionHistory() {
+    if (allPeoplePositionHistory != null) {
+      return CompletableFuture.completedFuture(allPeoplePositionHistory);
+    }
+    Map<String, Object> context = AnetObjectEngine.getInstance().getContext();
+    return AnetObjectEngine.getInstance().getPositionDao().getAllPositionHistory(context, uuid)
+        .thenApply(o -> {
+          allPeoplePositionHistory = o;
+          return o;
+        });
+  }
+
   public List<PersonPositionHistory> getPreviousPeople() {
     return previousPeople;
   }
 
+  @JsonIgnore
+  public List<PersonPositionHistory> getAllPeoplePositionHistory() {
```

Comment:
This method is only called by the tests, so really belongs somewhere under `src/test/java`.

Reply:
This method is moved under `src/test/java`.